src/constants/hotkeys.ts

```diff
@@ -5,6 +5,8 @@ const hotkeys = {
   SELECT_FOLDER: 'ctrl+o,command+o',
   REFRESH_FOLDER: 'ctrl+f5,command+f5',
   TOGGLE_LEFT_PANE: 'ctrl+b,command+b',
+  ZOOM_IN: 'ctrl+=,command+=',
+  ZOOM_OUT: 'ctrl+-,command+',
```

Comment:
great - please add to /docs/hotkeys.md also!

Reply:
Done!

Reply:
Done!